matrix_client/room.py

```diff
@@ -106,6 +106,24 @@ def send_emote(self, text):
         """
         return self.client.api.send_emote(self.room_id, text)
 
+    # See http://matrix.org/docs/spec/r0.0.1/client_server.html#m-file for the
```

Comment:
This is probably a bit excessive. I realise I did this below but that was probably wrong too :P. Having the link inside the documentation is enough.